python/pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[project]
+name = "intel-pyitt"
```

Comment:
should be `ittapi` by analogue with RUST naming schema for publishing

Reply:
changed to ittapi